shell/app/modules/application/pages/repo/repo-tree.tsx

```diff
@@ -295,6 +297,7 @@ const RepoTreePage = () => {
   const appDetail = appStore.useStore((s) => s.detail);
   const [info, tree, mode] = repoStore.useStore((s) => [s.info, s.tree, s.mode]);
   const hasAuth = usePerm((s) => s.app.externalRepo.edit.pass);
+  const [refType, setRefType] = React.useState();
```

Comment:
where is the usage of `refType`

Reply:
Nowhere was the refType used, it just triggers the component to rerender.